src/data/faq_de.json

```
@@ -376,6 +379,216 @@
+                                "textblock": [
+                                    "Nein, europaweit sind für Selbsttests keine digitalen Zertifikate vorgesehen. Testzertifikaten können nur für beaufsichtigte PCR-Tests (ausgenommen PoC-NAT PCR-Tests) und beaufsichtigte Schnelltests ausgestellt werden."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich für einen registrierten positiven Testbefund als Krankschreibung nutzen?",
+                                "anchor": "warn_without_tan_sickness_certificate",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, die Warnung anderer mit der neuen Funktion \"Tests erfassen und andere warnen\" führt gar nicht zur Anzeige eines positiven Testergebnisses in der Corona-Warn-App. Abgesehen davon ist die Anzeige in der CWA arbeitsrechtlich nicht zur Krankschreibung bzw. Arbeits-/Schulbefreiung zugelassen. Wenden Sie sich bitte an den Arzt Ihres Vertrauens oder an das zuständige Gesundheitsamt."
+                                ]
+                            },
+                            {
+                                "title": "Wie lange bleibt der Test in der App zu sehen?",
+                                "anchor": "warn_without_tan_in_app_display",
+                                "active": false,
+                                "textblock": [
+                                    "Die Warnung anderer mit der neuen Funktion \"Tests erfassen und andere warnen\" führt gar nicht zur Anzeige eines positiven Testergebnisses in der Corona-Warn-App.",
+                                    "Für Tests, die wie bisher in der App registriert werden, gilt nach wie vor: Tests stellen in der Diagnostik von Infektionskrankheiten immer nur eine Momentaufnahme dar, sie sind daher nicht unbegrenzt lange gültig. Nach 72 Stunden wird daher ein registrierter Test in der Corona-Warn-App als „veraltet“ grau dargestellt."
+                                ]
+                            },
+                            {
+                                "title": "Stoppt nach dem Teilen des positiven Ergebnisses meine Risikoermittlung? Wie lange?",
+                                "anchor": "warn_without_tan_risk_assessment",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, die Risikoermittlung und das Contact Tracing Protokoll werden nicht gestoppt. Die Warnung anderer mit den neuen Funktion \"Tests erfassen und andere warnen\" hat keine Auswirkungen auf die Risikoermittlung.",
+                                    "Wenn Sie sich trotz positiven Tests in der Öffentlichkeit bewegen, achten Sie bitte auf die Abstands- und Hygieneregeln und tragen Sie, wenn möglich, eine Maske zur Verringerung einer Infektionsgefahr für andere."

```

1. Shorten the title
2. Remove mention of "Contact Tracing Protokoll" and instead make clear what the app continues to do
3. "einer Infektionsgefahr" --> "der Infektionsgefahr"
4. (must do!) Change "Risikoermittlung" to in-app term & FAQ term "Risiko-Ermittlung"
5. Remove "neu" --> less maintenance intensive
```suggestion
                                "title": "Stoppt nach dem Teilen des positiven Ergebnisses meine Risiko-Ermittlung?",
                                "anchor": "warn_without_tan_risk_assessment",
                                "active": false,
                                "textblock": [
                                    "Nein, die Risiko-Ermittlung wird nicht gestoppt. Die Corona-Warn-App zeichnet also weiterhin ihre Kontakte auf und sendet Zufalls-IDs aus. Die Warnung anderer mit der Funktion \"Tests erfassen und andere warnen\" hat also keine Auswirkungen auf die Risiko-Ermittlung.",
                                    "Wenn Sie sich trotz positiven Tests in der Öffentlichkeit bewegen, achten Sie bitte auf die Abstands- und Hygieneregeln und tragen Sie, wenn möglich, eine Maske zur Verringerung der Infektionsgefahr für andere."
```

---

Thanks! https://github.com/corona-warn-app/cwa-website/pull/3322/commits/18de1f44bf14e01c9293fedf110eb4bc431732a1